numpy/lib/_function_base_impl.py

```diff
@@ -4417,7 +4282,10 @@ def quantile(a,
 
     Notes
     -----
-    In general, the quantile at probability level :math:`q` of a cumulative
+    Given a sample `a` from an underlying distribution, `quantile` provides a
```

Comment:
Don't we require many samples from the underlying distribution? For me `a sample` reads as a single value (usually a list is provided as the first input parameter of quantile)

Reply:
People use "sample" to mean both a single observation and a set of observations. Specifically to avoid ambiguity, I reserve the word "observation" for a single observation and "sample" for a set of observations.